packages/aws-fargate/images/instana-aws-fargate/Dockerfile-npm

```diff
@@ -1,23 +1,6 @@
-FROM node:12.22.7-buster AS instana-aws-fargate-build-nodejs
+FROM node:18-buster AS instana-aws-fargate-build-nodejs
```

Comment:
_thought:_ The tag `18-buster` will be updated everytime a new 18.x version comes out. That is, currently it refers to the same image as the `18.18.2-buster` tag. When Node.js 18.18.3 or 18.19.0 is released, it will point to that newer image. That means the outcome of our image build depends on the point in time when the build is executed. To have a reproducible build that does not depend on the time of execution, we might want to use a full version tag like 18.18.2-buster here. WDYT?

The same argument could be made for the Google cloud run image.

Reply:
Yeah I know, I thought. it doesn't really matter 🤔 Will concourse cache docker images at all? Or bootstrap a new fresh instance on each build?

Reply:
I will change it for the production use case.